### DISCUSSION TITLE
Welcome to restic-systemd-automatic-backup Discussions!

### DISCUSSION THREAD

**Discussion by erikw** [erikw/restic-automatic-backup-scheduler#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gerardbosch** [comment]
Hi there! Thanks @erikw for creating this!

I've been using it for a few years now, and I'm supper happy and relieved, as my backup is fully automated, unattended and virtually free, as the Backblaze B2 is very cheap. I'm using the script to make a daily backup only of my homedir, and B2 offers the first 10GB for free.

So if you put everything together, it's a fabulous backup solution for Linux!

**Reply by erikw** [reply]
I'm very glad that this solved the backup situation for you as well!

I've been using this setup myself also now for some 4-5 years, on several Linux, Mac and even Windows systems! (using cron and scheduled tasks for Mac and Windows respectively)
